coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,300 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    This aspect checks on the format of your code.
+    """
+    class docs:
+        example = """
+        # Here is an example of python code with lots of
+        # formatting issues including: trailing spaces, missing spaces
+        # around operators, uncorrect indentation etc...
+
+        x = {  'a':37,'b':42,
+
+        'c':927}
+
+        y = 'hello ''world'
+        z = 'hello '+'world'
+        a = 'hello {}'.format('world')
+        class foo  (     object  ):
+            def f    (self   ):
+                return       37*-+2
+            def g(self, x,y=42):
+                return y
+        def f  (   a ) :
+            return      37+-+a[42-x :  y**3]
```

Comment:
I think just doing one small example is better than having lots of stuff in here. Short and precise.